geomesa-spark/geomesa_pyspark/src/main/python/setup.py

```diff
@@ -5,5 +5,5 @@
     version='${python.version}',
     url='http://www.geomesa.org',
     packages=find_packages(),
-    install_requires=['pytz','pyspark']
+    install_requires=['pytz','pyspark>=2.1.1,<2.3.0']
```

Comment:
`pyspark` shouldn't be a dependency at all, it's included w/ Spark and the bootstrapping pulls in the version distributed via `SPARK_HOME` or it's equivalent.  There are differences w/ the pyspark module between versions and they are mated to spark versions, so this is generally dangerous.  It appears `pyspark` was added earlier with a "documentation-only" PR

Reply:
@aheyne what happens if we remove the dependency on pyspark here?